cartography/intel/aws/ec2/elastic_ip_addresses.py

```diff
@@ -32,14 +32,16 @@ def get_elastic_ip_addresses(boto3_session: boto3.session.Session, region: str)
 
 
 @timeit
-def transform_elastic_ip_addresses(elastic_ip_addresses: List[Dict]) -> List[Dict]:
+def transform_elastic_ip_addresses(elastic_ip_addresses: List[Dict], current_aws_account_id: str,) -> List[Dict]:
     addresses: List[Dict] = []
     for address in elastic_ip_addresses:
+        address['arn'] = f"arn:aws:ec2:{address.get('region')}:{current_aws_account_id}:eip/{address.get('AllocationId')}"
```

Comment:
Instead of `eip`, we should use `elastic-ip`.

ARN format for elastic ip should be `arn:${Partition}:ec2:${Region}:${Account}:elastic-ip/${AllocationId}`

Reference: https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonec2.html#amazonec2-elastic-ip